From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Matti=20Lehtim=C3=A4ki?= <matti.lehtimaki@gmail.com>
Date: Thu, 30 Aug 2018 19:54:16 +0300
Subject: [PATCH] (hybris) Fix actdead charging animation. MER#1949

---
 rootdir/init.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index e6c1ffed26d4fb05417597c8f42a98a028e98557..31c80442c67f91833d4546f5834c682105956bc8 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -1226,7 +1226,7 @@ on property:droid.late_start=trigger_late_start
     class_start late_start
 
 on charger
-    class_start charger
+    trigger late-init
 
 on property:sys.boot_completed=1
     bootchart stop
