From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Matti=20Lehtim=C3=A4ki?= <matti.lehtimaki@gmail.com>
Date: Thu, 30 Aug 2018 19:54:16 +0300
Subject: [PATCH] (hybris) Fix actdead charging animation. MER#1949

---
 rootdir/init.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index 388d5f10cbcc9a7ec9db6ed3fa41687106e3ca6f..731bf16d49c7f3b5c1119cc4236ad2a4b9970561 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -1202,7 +1202,7 @@ on property:droid.late_start=trigger_late_start
     class_start late_start
 
 on charger
-    class_start charger
+    trigger late-init
 
 on property:sys.boot_completed=1
     bootchart stop
