From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Matti Kosola <matti.kosola@jolla.com>
Date: Tue, 21 Feb 2017 13:13:26 +0100
Subject: [PATCH] (hybris) Disable usb config.

Signed-off-by: Matti Kosola <matti.kosola@jolla.com>
---
 rootdir/init.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index 31c80442c67f91833d4546f5834c682105956bc8..e4d248cc5ee9cb95cc88186b5bcaba0cc522107f 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -15,7 +15,7 @@ import /init.environ.rc
 #import /system/etc/init/hw/init.usb.rc
 import /init.${ro.hardware}.rc
 import /vendor/etc/init/hw/init.${ro.hardware}.rc
-import /system/etc/init/hw/init.usb.configfs.rc
+#import /system/etc/init/hw/init.usb.configfs.rc
 import /usr/libexec/droid-hybris/system/etc/init/hw/init.${ro.zygote}.rc
 
 # Cgroups are mounted right before early-init using list from /etc/cgroups.json
