From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Matti=20Lehtim=C3=A4ki?= <matti.lehtimaki@jolla.com>
Date: Mon, 10 Dec 2018 12:11:06 +0200
Subject: [PATCH] (hybris) Disable init_user0 which is not needed on Mer.

---
 rootdir/init.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index d5c516c868cb5f95a6a29a0870c88fa1de7e62a5..ec9cce0fcce16ec725a549cf73405ced92f6f5f3 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -1066,7 +1066,7 @@ on post-fs-data
 
     exec_start derive_sdk
 
-    init_user0
+    #init_user0
 
     # Set SELinux security contexts on upgrade or policy update.
     restorecon --recursive --skip-ce /data
