From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Matti=20Lehtim=C3=A4ki?= <matti.lehtimaki@jolla.com>
Date: Mon, 10 Dec 2018 12:11:06 +0200
Subject: [PATCH] (hybris) Disable init_user0 which is not needed on Mer.

---
 rootdir/init.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index adfaf3c856d4ffb047fc3e4eee29e92095d953b3..4278c7cab0789d065a06023094a0f58942245c28 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -1067,7 +1067,7 @@ on post-fs-data
 
     exec_start derive_sdk
 
-    init_user0
+    #init_user0
 
     # Set SELinux security contexts on upgrade or policy update.
     restorecon --recursive --skip-ce /data
