From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Matti=20Lehtim=C3=A4ki?= <matti.lehtimaki@gmail.com>
Date: Thu, 30 Aug 2018 19:54:16 +0300
Subject: [PATCH] (hybris) Fix actdead charging animation. MER#1949

---
 rootdir/init.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index e9b6e87165213949564deb29cdb31ad07fd27395..7956a384a8a53e857ad758777d6f4955d9218bd2 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -699,7 +699,7 @@ on property:droid.late_start=trigger_late_start
     class_start late_start
 
 on charger
-    class_start charger
+    trigger late-init
 
 on property:vold.decrypt=trigger_reset_main
     class_reset main
