From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Matti Kosola <matti.kosola@jolla.com>
Date: Tue, 21 Feb 2017 13:13:26 +0100
Subject: [PATCH] (hybris)[system/core] Disable usb config.

Signed-off-by: Matti Kosola <matti.kosola@jolla.com>
---
 rootdir/init.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index cfe5ae303158f49927099b7edf9dd9e7e06f5a66..7427aed7a52b5b865d7ec30f2f672db3041c949a 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -15,7 +15,7 @@ import /init.environ.rc
 #import /init.usb.rc
 import /init.${ro.hardware}.rc
 import /vendor/etc/init/hw/init.${ro.hardware}.rc
-import /init.usb.configfs.rc
+#import /init.usb.configfs.rc
 import /init.${ro.zygote}.rc
 
 # Cgroups are mounted right before early-init using list from /etc/cgroups.json
