From 25929384dc9482df6671dc5da24c87b3fd32f1dd Mon Sep 17 00:00:00 2001
From: Jussi Laakkonen <jussi.laakkonen@jolla.com>
Date: Wed, 31 Jan 2018 17:12:14 +0200
Subject: [PATCH 34/39] (hybris) Remove /sbin from droid PATH.

---
 rootdir/init.environ.rc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rootdir/init.environ.rc.in b/rootdir/init.environ.rc.in
index 516dc82f6..90c2b64d6 100644
--- a/rootdir/init.environ.rc.in
+++ b/rootdir/init.environ.rc.in
@@ -1,7 +1,7 @@
 # set up the global environment
 on init
 
-    export PATH /sbin:/vendor/bin:/system/sbin:/system/bin:/system/xbin
+    export PATH /vendor/bin:/system/sbin:/system/bin:/system/xbin
     # This is not 64-bit safe -stskeeps
     # Hopefully now it is - ghosalmartin
     # this breaks mixed 32/64-bit devices -mal
-- 
2.17.1

