From 2041529d9c5666862e174151dfa9df9d4ae9e7ce Mon Sep 17 00:00:00 2001
From: Jussi Laakkonen <jussi.laakkonen@jolla.com>
Date: Wed, 31 Jan 2018 17:12:14 +0200
Subject: [PATCH 34/50] (hybris) Remove /sbin from droid PATH.

---
 rootdir/init.environ.rc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rootdir/init.environ.rc.in b/rootdir/init.environ.rc.in
index 73fbaa413..8f95df9c6 100644
--- a/rootdir/init.environ.rc.in
+++ b/rootdir/init.environ.rc.in
@@ -1,6 +1,6 @@
 # set up the global environment
 on early-init
-    export PATH /sbin:/vendor/bin:/system/sbin:/system/bin:/system/xbin
+    export PATH /vendor/bin:/system/sbin:/system/bin:/system/xbin
     # this breaks mixed 32/64-bit devices -mal
     # for 32-bit devices with android kernel logger there is now a file called init.extraenv.armeabi-v7a.rc - krnlyng
     # export LD_LIBRARY_PATH /usr/libexec/droid-hybris/lib64-dev-alog:/vendor/lib64:/system/lib64:/usr/libexec/droid-hybris/lib-dev-alog:/vendor/lib:/system/lib
-- 
2.23.0

