From 8ab5e607e71ac798d0a8b2045492a55d6b273534 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Matti=20Lehtim=C3=A4ki?= <matti.lehtimaki@jolla.com>
Date: Mon, 10 Dec 2018 12:11:06 +0200
Subject: [PATCH 38/40] (hybris) Disable init_user0 which is not needed on Mer.

---
 rootdir/init.rc | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index bc6b3e9d0..66aaf86ce 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -551,7 +551,8 @@ on post-fs-data
     mkdir /data/cache/backup_stage 0700 system system
     mkdir /data/cache/backup 0700 system system
 
-    init_user0
+    # Requires vold which is is not needed in Mer
+    #init_user0
 
     # Set SELinux security contexts on upgrade or policy update.
     restorecon --recursive --skip-ce /data
-- 
2.17.1

