From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Matti=20Lehtim=C3=A4ki?= <matti.lehtimaki@jolla.com>
Date: Mon, 10 Dec 2018 12:11:06 +0200
Subject: [PATCH] (hybris) Disable init_user0 which is not needed on Mer.

---
 rootdir/init.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index 5d5468262983979605825f6457206336a246f444..c5e71291d517780873311e39cbbebcb450a8f624 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -605,7 +605,7 @@ on post-fs-data
     wait_for_prop apexd.status ready
     parse_apex_configs
 
-    init_user0
+    #init_user0
 
     # Set SELinux security contexts on upgrade or policy update.
     restorecon --recursive --skip-ce /data
